tests/integration/test_render.py

```diff
@@ -32,18 +33,22 @@ class TestDragAndDropRender(BaseIntegrationTest):
     PAGE_TITLE = 'Drag and Drop v2'
     PAGE_ID = 'drag_and_drop_v2'
     ITEM_PROPERTIES = [{'text': '1'}, {'text': '2'}, {'text': 'X'}, ]
+    SIDES = ['Top', 'Bottom', 'Left', 'Right']
 
-    def load_scenario(self, item_background_color="", item_text_color=""):
+    def load_scenario(self, item_background_color="", item_text_color="", zone_labels=False, zone_borders=False):
+        json = load_resource("integration/data/test_data_a11y.json")
```

Comment:
@itsjeyd I did have this as "data", but pylint complains:

    $ pylint drag_and_drop_v2 tests
    ************* Module tests.integration.test_render
    W: 39,8:TestDragAndDropRender.load_scenario: Redefining name 'data' from outer scope (line 1)

Any suggestions?